### DISCUSSION TITLE
add sites from group interface

### DISCUSSION THREAD

**Discussion by johanpeters** [akeeba/panopticon#429] [Ideas]
Currently the only way to add sites to a group is to edit every specific site and add them to a group.
it would be easier to have a option in the group configuration to edit / create groups and add one or multiple sites to the group. this would be a lot easer .

**Comment by nikosdion** [comment]
The group configuration would be the wrong place to do that. Conceptually, this belongs to sites management.

The only problem is that groups are configuration options, values inside a JSON field. I am not sure how efficient that would be, and whether it would cause issues such as timeouts. I have to take a closer look before committing to it.

**Reply by nikosdion** [reply]
Tracking with https://github.com/akeeba/panopticon/issues/480
